frontend/src/components/SignInUpModal.js

```diff
@@ -0,0 +1,317 @@
+import React, { Component } from 'react';
+import ReactModalLogin from 'react-modal-login';
+import ReactDOM from 'react-dom';
+import { fb } from '../App';
+import firebase from 'firebase';
+import { changeMapState } from "../actions/index";
+import { connect } from "react-redux";
+import zxcvbn from 'zxcvbn';
+import tldjs from 'tldjs';
+
+const { getDomain, getPublicSuffix } = tldjs
+
+function mapDispatchToProps(dispatch) {
+  return {
+    changeMapState: mapState => dispatch(changeMapState(mapState))
+  };
+}
+
+const mapStateToProps = state => {
+  return { articles: state.articles };
+}
+
+class LogInAndSignUp extends Component {
+  constructor(props) {
+    super(props);
+
+    this.reduxState = this.props.articles[0];
+
+    this.state = {
+      loggedIn: false,
+      showModal: true,
+      loading: false,
+      error: null,
+      initialTab: 'login',
+      recoverPasswordSuccess: null,
+      credentials: null,
+    };
+  }
+
+  updateRedux(userCredentials, userLoggedIn) {
+    console.log('updating redux')
+    console.log(this.reduxState)
```

Comment:
Remove console logs here, api keys can be shown

Reply:
I'm going to remove all the console.logs